tube/mappings/subject.yaml

```diff
@@ -0,0 +1,49 @@
+subject:
```

Comment:
let's move this file to unit tests folder, and specify mapping yaml as a configuration filepath for the class